provisioning/roles/base/tasks/ntp.yml

```diff
@@ -31,3 +31,34 @@
                   {{ groups['infrastructure'] }}
                 {%- endif -%}"
     ntp_servers: "{{ common_ntp_servers }}"
+
+# We should probably switch to the system default of chronyd instead of
+# insisting on ntpd.
+- block:
+  - name: Disabling chronyd service
+    systemd:
+      name: chronyd.service
+      state: stopped
+      enabled: False
+
+  - name: Disabling chrony-wait service
+    systemd:
+      name: chrony-wait.service
+      state: stopped
+      enabled: False
+
+  - name: Enabling ntpd service
+    systemd:
+      name: ntpd.service
+      state: started
+      enabled: True
```

Comment:
Does this actually work on newer Fedoras?  I thought ntpd went away in Fedora35?

Reply:
As relayed in standup this is part of a block of tasks that is not run if the system being provisioned does not have ntpd.  It may still not run based on other conditions, but lack of ntpd prevents it.

Reply:
This may be something we want to clean up in the future.  Potentially dropping ntpd support altogether.  I won't let it hold up this change, though.